src/dapp/src/shared/csvSigner.ts

```diff
@@ -25,12 +25,15 @@ export async function signCsv(csv: string, includeWeighting: boolean): Promise<{
     results.data
       .filter((row) => !!row.address)
       .map((row): Promise<SignedCsv> => {
-        const publicKey = algo.decodeAddress(row.address).publicKey
+        let publicKey = algo.decodeAddress(row.address).publicKey
```

Comment:
If you chose NO_WEIGHTING and uploaded a CSV with weighting and one of them was invalid then this failed so making a change to only process weighting when it's needed